From: Matthias Clasen <mclasen@redhat.com>
Date: Tue, 15 Jan 2019 13:45:24 -0500
Subject: docs: Clarify that --command is only for run

This may not otherwise be obvious.

Closes: #2592
Approved by: matthiasclasen
Applied-upstream: 1.1.4, commit:81b25058ae707cbe20d21754ebde74206fb45be9
---
 doc/flatpak-build-finish.xml | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/doc/flatpak-build-finish.xml b/doc/flatpak-build-finish.xml
index 2728302..0754335 100644
--- a/doc/flatpak-build-finish.xml
+++ b/doc/flatpak-build-finish.xml
@@ -89,6 +89,11 @@
                     The command to use. If this option is not specified,
                     the first executable found in <filename>files/bin</filename>
                     is used.
+                </para><para>
+                    Note that the command is used when the application is run
+                    via <command>flatpak run</command>, and does not affect what
+                    gets executed when the application is run in other ways,
+                    e.g. via the desktop file or D-Bus activation.
                 </para></listitem>
             </varlistentry>
 
